File: collectors/openbus.ping/src/main/java/br/com/produban/openbus/ping/Launcher.java
package br.com.produban.openbus.ping;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.cache.annotation.EnableCaching;
import org.springframework.scheduling.annotation.EnableScheduling;

@SpringBootApplication
@EnableScheduling
@EnableCaching
public class Launcher {

	public static void main(String[] args) {
		SpringApplication.run(Launcher.class, args);
	}

}

File: collectors/openbus.f5box/src/main/scala/br/com/produban/openbus/f5/Launcher.scala
package br.com.produban.openbus.f5

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.cache.annotation.EnableCaching
import org.springframework.scheduling.annotation.EnableScheduling

@SpringBootApplication
@EnableScheduling
@EnableCaching
class LauncherConfiguration

object Launcher extends App {
  SpringApplication.run(classOf[LauncherConfiguration])
}
